api/src/services/verification-service.ts

```diff
@@ -19,12 +19,12 @@ import { JsonldGenerator } from '../utils/jsonld';
 import { Subject } from '../models/types/verification';
 import { ILogger } from '../utils/logger';
 import { ILock, Lock } from '../utils/lock';
+import { SERVER_IDENTITY } from '../config/server';
```

Comment:
this should be part of the constructor parameter as for instance `verificationServiceConfig: VerificationServiceConfig`

Reply:
^ makes it easier to test without always setting the global variable `SERVER_IDENTITY` but by simply using the config added as parameter to the constructor

Reply:
The problem is in the global variable verificationService created in src/routers/services.ts
Unlikely all those global variables make impossible to have this information at instantiation time since the SERVER_IDENTITY is taken from the database.
A way to solve this problem would be to have a service that is responsible to get (and cache) this value once is readed from database.